### PR TITLE
 fix missing other language

### PR DESCRIPTION
**Pull request by Una-p**
Im so sorry for it. fix them missing line at #1681

### CODE REVIEW

**Comment by Una-p** [comment]
I saw your commit. thanks, but we have to fix 545 583 585 625 line, too.

File: ui/raidboss/data/02-arr/raid/t9.js [hunk 1]
```diff
@@ -542,7 +541,7 @@
         'Divebomb': 'ダイブボム',
         'Fireball': 'ファイアボール',
         'Ghost': 'ゴースト',
-        'Heavensfall': 'ヘヴンスフォール',
+        'Heavensfall': '天地崩壊',
```
Comment (panicstevenson) [inline ui/raidboss/data/02-arr/raid/t9.js:544]: Was this one intentional?

Comment (Una-p) [reply ui/raidboss/data/02-arr/raid/t9.js:544]: That's not what I did. I just followed last version what i didn't touched

**Comment by quisquous** [comment]
Aaaaaaaaah sorry.  I think I misunderstood what was going on.  I shouldn't merge patches when I'm this tired.  Thank you for following up here.

**Comment by Una-p** [comment]
Nevermind, it's all my fault. I'll try to check four times before request. XD

**Pull request merged**